vitest.shared.config.ts

```diff
@@ -13,8 +13,10 @@ export default defineConfig({
       toFake: ["setTimeout", "Date"],
     },
     watch: false,
+    include: ["test/**/*.spec.ts"],
```

Comment:
Just to force the issue with ESM, we should probably rename these files to a `.mts` extension.

Reply:
renaming all the test files?